samples/PSFScriptingSample/readme.md

```diff
@@ -0,0 +1,13 @@
+# Scripting sample
+This solution contains example scripts and configurations to make an application to see how scripting works in the Package Support Framework.
+
+## How to make the package?
+To make and install the sample application all you need to do is rebuild the solution from inside Visual Studio.  The build system will package the app and install it on your computer.
```

Comment:
>make [](start = 3, length = 4)

Do you need to run the solution in the debugger to get the app to deploy on the system? Suggest the following:

Open solution in Visual Studio and build the solution. To deploy the package...

Reply:
Building does deploy the package.  All the user needs to do is run msbuild on the solution.  The csproj file tells msbuild to build the solution, make the app, and deploy the app.

Reply:
Ok cool! I would still prefer the change:
Open solution in Visual Studio and build the solution. The build process will package and deploy the test applications to your system.

---
In reply to: [357375637](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/106#discussion_r357375637) [](ancestors = 357375637)

Reply:
You can remove one of these lines.

---
In reply to: [357390799](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/106#discussion_r357390799) [](ancestors = 357390799,357375637)